docs/content/reference/versioning-support.md

```diff
@@ -11,18 +11,18 @@ There are different components that might require independent versioning and sup
 
 ### HO
 
-- A HO is intended to be released for each OCP release.
-- A HO released for OCP N minor release must support N, N-1 and N-2 OCP minor releases.
-- A HO release if anything, will only do best effort to support future OCP versions.
+- A HO is intended to be released for each OCP minor release.
+- A HO released for OCP N minor release must support guest clusters of N, N-1 and N-2 OCP minor releases.
```

Comment:
maybe s/guest clusters of/Hosted Clusters with a release image version of"

Reply:
yep, fixed